src/QsCompiler/Transformations/ContentLifting.cs

```diff
@@ -347,6 +351,14 @@ public override QsSpecialization OnControlledSpecialization(QsSpecialization spe
                 return rtrn;
             }
 
+            public override QsSpecialization OnControlledAdjointSpecialization(QsSpecialization spec)
```

Comment:
The logic for the lifting, at least as of now, doesn't actually need to know when it is in this specialization. But I guess it could be nice to have more complete information. I'll leave it up to you if you want to include this and the associated flag.

Reply:
Yes, I realized that looking at what queries these handles. It just felt like such an implementation detail that you want to know what specialization you are in except if it's a controlled adjoint speciliazation.